packages/http/src/validator/deserializers/index.ts

```diff
@@ -11,13 +11,14 @@ import type { IHttpNameValue, IHttpNameValues } from '../../';
 export const header: Dictionary<deserializeFn<IHttpNameValue>, 'simple'> = { simple };
 export const query: Dictionary<
   deserializeFn<IHttpNameValues>,
-  'form' | 'spaceDelimited' | 'pipeDelimited' | 'commaDelimited' | 'deepObject'
+  'form' | 'spaceDelimited' | 'pipeDelimited' | 'commaDelimited' | 'deepObject' | 'simple'
 > = {
   form,
   spaceDelimited: delimited('%20'),
   pipeDelimited: delimited('|'),
   commaDelimited: delimited(','),
   deepObject,
+  simple: form,
```

Comment:
query parameters are now able to have style `simple` with the new http-spec. Needed to provide a deserializer for that style. @P0lip thinks this may be a bug, but in the meantime, we can use the `form` deserializer if we receive `style` of `simple`